buildSrc/build.gradle.kts

```diff
@@ -4,4 +4,9 @@ plugins {
 
 repositories {
     jcenter()
+    google()
+}
+
+dependencies {
+    implementation("com.android.tools.build:gradle:3.3.1")
```

Comment:
Is there a new version of this now that Android Studio 3.4 has been released?

Reply:
this would be updated in a separate PR to update all the dependency versions. This would use the same version as the one in the top level build.gradle. Unfortunately, there isn't a simple way to share variables between the two files.

Reply:
Sharing can be done in groovy version of `build.gradle`. It is because of it being a Kotlin build-file that these vars can no longer be easily shared?

Reply:
I'm not sure if that's possible even in groovy. This is the buildSrc gradle file, which is kinda considered a separate project from rest of the code/modules. One possible way to share is through the gradle.properties file, but it might not be worth the extra work/quirk just to share one variable

Reply:
Ah, yes, you're correct. It is part of `buildSrc`... I forgot to realize that.